### PR TITLE
 expose cesium viewer

### LINKED ISSUE #77: cesiumservice usage
hello , youve got a great product !
my problem is 
there is no way to inject CesiumService and use the getViewer & getScene functions in a component/service which is located out side of ac-map in the dom hirarchy

### PR DESCRIPTION
**Pull request by eitanfr**
resolve #77

### CODE REVIEW

File: demo/app/components/demo-map/demo-map.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<ac-map [enableTilt]="false">
+<ac-map [enableTilt]="false" id="main-map">
```
Comment (tomermoshe) [inline demo/app/components/demo-map/demo-map.component.html]: delete id

File: src/components/ac-map/ac-map.component.ts [hunk 2]
```diff
@@ -61,6 +67,13 @@ export class AcMapComponent implements OnChanges, OnInit {
 	 */
 	@Input()
 	enableTilt: boolean = AcMapComponent.DEFAULT_TILT_ENABLE;
+ 
+	/**
+	 * Set the id name of the map
+	 * default: 'default-[index]'
```
Comment (tomermoshe) [inline src/components/ac-map/ac-map.component.ts]: fix doc - no default

File: demo/app/components/demo-map/demo-map.component.html (hunk 1 above)
Comment (eitanfr) [inline demo/app/components/demo-map/demo-map.component.html]: k

File: src/components/ac-map/ac-map.component.ts [hunk 3]
```diff
@@ -10,11 +10,17 @@ import { DynamicPolylineDrawerService } from '../../services/drawers/dynamic-pol
...
+import { ViewersManagerService } from '../../services/viewers-service/viewers-manager.service';
 
 /**
  * This is a map implementation, creates the cesium map.
  * Every layer should be tag inside ac-map tag
  *
+ * Accessing cesium viewer:
+ * 1. acMapComponent.getViewer()
+ * 2. Use ViewerManagerService.getCesiumViewer(mapId).
+ * 		mapId auto-generated string: 'map-[index]'
```
Comment (tomermoshe) [inline src/components/ac-map/ac-map.component.ts]: fix comment

**Pull request merged**